frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDetailsActions.tsx

```diff
@@ -78,6 +90,32 @@ const PipelineRunDetailsActions: React.FC<PipelineRunDetailsActionsProps> = ({ o
               ) : (
                 <React.Fragment key="compare-runs" />
               ),
+              !isRunActive ? (
+                <Tooltip
+                  position="left"
+                  content={
+                    <div>
+                      Archived runs cannot be restored until its associated experiment is restored.
+                    </div>
+                  }
+                >
+                  <DropdownItem
+                    isDisabled={!isRunActive}
```

Comment:
this should be disabled if the experiment is archived. so you would need to do this
```ts
  const [experiment] = useExperimentById(experimentId);
  const isExperimentActive = experiment?.storage_state === StorageStateKF.AVAILABLE;
```

Then the tooltip should only be displayed if the dropdown item is disabled